packages/core/src/lib/markdown-it/plugins/custom-component/inlineTags.js

```diff
@@ -0,0 +1,78 @@
+
+/**
+ * According to markdown spec, all non-block html tags are treated as "inline"
+ * tags (wrapped with <p></p>), including those "unknown" tags
+ *
+ * Therefore, markdown-it processes "inline" tags and "unknown" tags in the same
+ * way, and does not care if a tag is "inline" or "unknown"
+ *
+ * As we want to take those "unknown" tags as custom components, we should
+ * treat them as "block" tags
+ *
+ * So we have to distinguish between "inline" and "unknown" tags ourselves
+ *
+ * The inline tags list comes from MDN
+ *
+ * @see https://spec.commonmark.org/0.29/#raw-html
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements
+ */
+const inlineTags = [
```

Comment:
Would it be possible to whitelist `pic / thumbnail` as well? https://github.com/MarkBind/markbind/pull/1534#pullrequestreview-637316649
So its consistent with `<img />` tags (e.g. author may add `float` style to float the image to the right of paragraph)

The components would likely have to be adapted slightly in the ssr pr then in terms of just inline tags.